hack/update-codegen-dockerized.sh

```diff
@@ -36,40 +36,52 @@ ANTREA_PKG="antrea.io/antrea"
-    "pkg/controller/networkpolicy EndpointQuerier testing"
-    "pkg/controller/querier ControllerQuerier testing"
-    "pkg/flowaggregator/exporter Interface testing"
-    "pkg/ipfix IPFIXExportingProcess,IPFIXRegistry,IPFIXCollectingProcess,IPFIXAggregationProcess testing"
-    "pkg/ovs/openflow Bridge,Table,Flow,Action,CTAction,FlowBuilder,Group,BucketBuilder,PacketOutBuilder testing"
-    "pkg/ovs/ovsconfig OVSBridgeClient testing"
-    "pkg/ovs/ovsctl OVSCtlClient testing"
-    "pkg/querier AgentNetworkPolicyInfoQuerier,AgentMulticastInfoQuerier testing"
-    "pkg/flowaggregator/querier FlowAggregatorQuerier testing"
-    "third_party/proxy Provider testing"
-  )
+MOCKGEN_DEFAULT_TARGETS=(
+  "pkg/agent/cniserver/ipam IPAMDriver testing"
+  "pkg/agent/flowexporter/connections ConnTrackDumper,NetFilterConnTrack testing"
+  "pkg/agent/interfacestore InterfaceStore testing"
+  "pkg/agent/memberlist Memberlist testing"
+  "pkg/agent/multicast RouteInterface testing"
+  "pkg/agent/types McastNetworkPolicyController testing"
+  "pkg/agent/nodeportlocal/portcache LocalPortOpener testing"
+  "pkg/agent/nodeportlocal/rules PodPortRules testing"
+  "pkg/agent/openflow Client,OFEntryOperations testing"
+  "pkg/agent/proxy Proxier testing"
+  "pkg/agent/querier AgentQuerier testing"
+  "pkg/agent/route Interface testing"
+  "pkg/agent/ipassigner IPAssigner testing"
+  "pkg/agent/secondarynetwork/podwatch InterfaceConfigurator testing"
+  "pkg/agent/secondarynetwork/ipam IPAMDelegator testing"
+  "pkg/agent/util/ipset Interface testing"
+  "pkg/agent/util/iptables Interface testing mock_iptables_linux.go" # Must specify linux.go suffix, otherwise compilation would fail on windows platform as source file has linux build tag.
+  "pkg/agent/util/netlink Interface testing mock_netlink_linux.go"
+  "pkg/antctl AntctlClient ."
+  "pkg/controller/networkpolicy EndpointQuerier testing"
+  "pkg/controller/querier ControllerQuerier testing"
+  "pkg/flowaggregator/exporter Interface testing"
+  "pkg/ipfix IPFIXExportingProcess,IPFIXRegistry,IPFIXCollectingProcess,IPFIXAggregationProcess testing"
+  "pkg/ovs/openflow Bridge,Table,Flow,Action,CTAction,FlowBuilder,Group,BucketBuilder,PacketOutBuilder testing"
+  "pkg/ovs/ovsconfig OVSBridgeClient testing"
+  "pkg/ovs/ovsctl OVSCtlClient testing"
+  "pkg/querier AgentNetworkPolicyInfoQuerier,AgentMulticastInfoQuerier testing"
+  "pkg/flowaggregator/querier FlowAggregatorQuerier testing"
+  "third_party/proxy Provider testing"
+)
+
+MOCKGEN_MULTICLUSTER_TARGETS=(
+  "multicluster/controllers/multicluster/leader MemberClusterStatusManager . mock_membercluster_status_manager.go"
+  "multicluster/controllers/multicluster/commonarea CommonArea,RemoteCommonArea,ImportReconciler,RemoteCommonAreaGetter . mock_remote_common_area.go"
+)
+
+MOCKGEN_TARGETS=("${MOCKGEN_DEFAULT_TARGETS[@]}" "${MOCKGEN_MULTICLUSTER_TARGETS[@]}")
 
+# Generate mocks for testing with mockgen.
+function generate_mocks {
```

Comment:
I was thinking could we include this script from `multicluster/hack/update-codegen-dockerized.sh`, and call generate_mocks() there, so still multicluster related code is in the `multicluster` directory. What you think about that approach?

Reply:
I feel both works for me, let's me add the file back and share the common codes.

Reply:
Done